include/dpp/cluster.h

```diff
@@ -527,7 +596,7 @@ class DPP_EXPORT cluster {
 	 * The function signature for this event takes a single `const` reference of type button_click_t&, and returns void.
 	 */
 	event_router_t<button_click_t> on_button_click;
-	
+
```

Comment:
your editor has been doing automatic tidyips of the file, you should probably turn this feature off as it will make it harder to review your prs